core/bin/migrations/20241204_create_nodes.rs

```diff
@@ -0,0 +1,336 @@
+use anyhow::{anyhow, Result};
+use base64::prelude::BASE64_STANDARD_NO_PAD;
+use base64::Engine;
+use clap::Parser;
+use dust::stores::{postgres, store};
+use dust::utils;
+use futures::future::try_join_all;
+use lazy_static::lazy_static;
+use regex::Regex;
+
+#[derive(Parser, Debug)]
+struct Args {
+    #[arg(long, default_value = "false")]
+    execute: bool,
+
+    #[arg(long, default_value = "1000")]
+    batch_size: i64,
+
+    #[arg(long)]
+    data_source: Option<i64>,
+}
+
+#[derive(Clone, Copy, Debug)]
+enum NodeType {
+    Document,
+    Table,
+}
+
+lazy_static! {
+    static ref DOC_MIME_TYPES_MATCH: Vec<(fn(&str) -> bool, &'static str)> = vec![
+        // Confluence
+        ((|s: &str| s.starts_with("confluence-page-")), "application/vnd.dust.confluence-page"),
```

Comment:
As discussed IRL would be nice to align all our mimeTypes to `application|text/vnd.dust.{platform}.{thing}`